Documentation/proposal-runtime_versioning.md

```diff
@@ -0,0 +1,21 @@
+# Proposal for runtime versioning
+
+Capstan runtime support currently supports runtime name (e.g. node) in meta/run.yaml. But there are many versions of NodeJS available (e.g. 4.4.5, 6.2.2, ...). At the moment, a new runtime name is required for each version.
+
+I propose the following solution:
+
+We add a new attribute to the meta/run.yaml, `version`:
+```yaml
+# meta/run.yaml
+
+# required run attributes
+runtime: node
+version: 4.4.5   # <--------------------- new attribute
+
+# runtime-specific run attributes
+main: app.js
+```
+
+And the Capstan would then simply join the runtime string and version string as `{runtime}_v{version}` to get the actual runtime name, e.g. `node_v4.4.5`.
```

Comment:
Make sure we don't forget to issue an explicit warning when a particular version has not been found in a local or remote package hub.